client/blocks/subscription-length-picker/docs/example.jsx

```diff
@@ -0,0 +1,46 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getPlan } from 'lib/plans';
+import { PLAN_BUSINESS, PLAN_BUSINESS_2_YEARS } from 'lib/plans/constants';
+import { SubscriptionLengthPicker } from 'blocks/subscription-length-picker';
+import PropTypes from 'prop-types';
+
+const productsWithPrices = [
+	{
+		planSlug: PLAN_BUSINESS,
+		plan: getPlan( PLAN_BUSINESS ),
+		priceFull: 200,
+		priceMonthly: 200 / 24,
+	},
+	{
+		planSlug: PLAN_BUSINESS_2_YEARS,
+		plan: getPlan( PLAN_BUSINESS_2_YEARS ),
+		priceFull: 380,
+		priceMonthly: 380 / 48,
+	},
+];
+
+const SubscriptionLengthPickerExample = () => (
+	<div>
+		<SubscriptionLengthPicker
+			initialValue=""
+			currencyCode="USD"
+			productsWithPrices={ productsWithPrices }
+			initialValue={ PLAN_BUSINESS_2_YEARS }
+			translate={ x => x }
+		/>
+	</div>
+);
+
+SubscriptionLengthPickerExample.displayName = 'SubscriptionLengthPicker';
```

Comment:
👍 Note that we only need this in devdocs to avoid us mangling display names in prod. Eg shorteners might change this to `t` otherwise in production.

Reply:
Shorteners rarely mangle property names such as `displayName`, in our case they don't - it would be asking for trouble. But yes, this is only used in devdocs.

Reply:
It happens in devdocs which is why I was explaining why its needed there and nowhere else :) See for example https://github.com/Automattic/wp-calypso/pull/14481